### PR TITLE
Make bs col classes in challenge request table consistent

### LINKED ISSUE #2992: Challenge request and budget tables weird width
First column is far too wide.

Screenshot 2023-08-22 at 16 52 12

### PR DESCRIPTION
**Pull request by amickan**
Closes #2992

### CODE REVIEW

**Comment by jmsmkn** [comment]
We also see this in the budget table, does this solve that too? 

Screenshot 2023-11-06 at 14 28 17

**Comment by jmsmkn** [comment]
Isn't it better to use tables for both, not set column widths and then let the browser handle it?

**Comment by amickan** [comment]
Oh true, that should work.

File: app/grandchallenge/challenges/templates/challenges/partials/budget_table.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
     <table class="table table-hover table-sm">
         {% for field, value in object.budget.items %}
             <tr {% if "Total" in field %}class="table-light"{% endif %}>
-                <td class="col-5 font-weight-bold">{{ field }}</td>
-                <td class="col-7">{{ value }} €</td>
+                <td class="font-weight-bold">{{ field }}</td>
+                <td>{{ value }} €</td>
```
Comment (jmsmkn) [inline app/grandchallenge/challenges/templates/challenges/partials/budget_table.html:19]:
```suggestion
                <td>€&nbsp;{{ value }}</td>
```

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**